String/test/CheckHeterogram.test.js

```diff
@@ -0,0 +1,38 @@
+import { checkHeterogram } from '../CheckHeterogram'
+
+describe('Testing checkHeterogram', () => {
+  it('expects to throw type error if given a non string argument', () => {
+    expect(() => checkHeterogram(0)).toThrow()
+  })
+  it('expects to throw type error if given a non string argument', () => {
+    expect(() => checkHeterogram(true)).toThrow()
+  })
+  it('expects to return true if the argument is a heterogram', () => {
+    const SUT = checkHeterogram('hero')
```

Comment:
please get rid of the `SUT` variable in all tests - it adds no value

Reply:
Thank you for noticing this detail. It was my first time implementing testing, so clearly made a quite foolish error looking back at it now.